coalib/misc/Caching.py

```diff
@@ -1,8 +1,8 @@
 import time
 import os
+import logging
```

Comment:
ah good point, yeah then let's file newcomer issues :)
(but not for the cases where you have screwed up the alphabetical order :P)